db/migrate/20240812105008_add_latlong_to_neighbour.rb

```diff
@@ -0,0 +1,5 @@
+class AddLatlongToNeighbour < ActiveRecord::Migration[7.1]
+  def change
+    add_column :neighbours, :latlong, :jsonb
```

Comment:
Is it not worth using rgeo here for the way Leaflet wants to consume coordinates?
I think on planning application we store this as `factory.point(longitude, latitude)`